upgrade/upgrade-1.1.0.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen PrestaShop plugin
+ *
+ * Copyright (c) 2019 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+/**
+ * This function is automatically called on version upgrades.
+ *
+ * Version 1.1.0 introduces refund feature
+ *
+ * @param Adyen $module
+ * @return bool
+ */
+function upgrade_module_1_1_0(Adyen $module)
+{
+    return $module->registerHook('actionOrderSlipAdd');
```

Comment:
```suggestion
   $module->registerHook('actionOrderSlipAdd');
   $module->registerHook('displayPaymentTop');
   return true;
```

Reply:
I believe the best way to go about this is for you to add it to your repository as version 1.1.1 or something